app/services/config_report_importer.rb

```diff
@@ -1,4 +1,13 @@
 class ConfigReportImporter < ReportImporter
+  DEFAULT_REPORT_SCANNER = [
```

Comment:
Could this use some plugin registry instead of creating new one in this class? Otherwise I think it should be moved under app/registries to avoid auto-loader issue, see https://github.com/theforeman/foreman/blob/develop/app/registries/README.md for more info

Reply:
Thanks for pointing me at registries! I'll change this to have a registry for adding report scanner that `ReportImporter` can call.